README.md

```diff
@@ -71,26 +71,29 @@ Being able to write SQL queries is an important skill, but being able to design
 
 It is recommended that you complete these specs in order. The LeetCode exercises build on the **SQL Foundations** exercises.
 
+### DAY 1-2 
 **SQL Foundations**
 - [ ] [SQLBolt](https://sqlbolt.com/) exercises completed
```

Comment:
if you put x's in these they render to checked boxes, easier to track progress
`- [X]`